packages/api/src/modules/phrase/types.ts

```diff
@@ -0,0 +1,15 @@
+import { gql } from 'apollo-server-express';
+
+export default gql`
+  type Phrase {
+    _id: ID!
+    file: File!
```

Comment:
Tem uma coisa que não sei se entendi no graph-ql direito. Aqui por exemplo, você tá falando que dentro de Phrase tem o campo "file", que é do tipo "File", e esse tipo File (definido em outro módulo) é um conjunto de outros campos, beleza.
Só que nas linhas mais abaixo, você colocou que sua mutation vai retornar uma Phrase, logo também deveria retornar esse campo file como um conjunto de campos referente a File, beleza.
Porém, se você for lá no seu arquivo do resolver, você tá retornando "phrase", que é tipo um documento do mongoDB. Teoricamente, nesse documento, temos "file" apenas como um id, e não como um conjunto de outros campos, conseguiu captar minha dúvida?
Tipo, eu não sei se isso é alguma inconsistência, ou se vai estourar um erro, ou se o mongoose de alguma forma entende que o "phrase" é para ser retornado não apenas com uma referência do file, mas todo o document referente à file.

@UriellViana me ajuda a entender isso

Reply:
Aqui deveria ser um ID, mas pode existir outro tipo PhraseWithFile que aí sim usa o tipo File pra retorno.